.github/workflows/build.yml

```diff
@@ -23,7 +23,13 @@ jobs:
               components: rustfmt
 
          - uses: actions/checkout@v2
-         - name: Check
-           run: cargo check
+         - name: build
+           run: cargo build --release
+           run: strip target/release/eww
+           - uses: actions/upload-artifact@v2
+             with:
+                name: eww-binary
+                path: target/
+
```

Comment:
This won't be that simple, i'm afraid, as a release binary should be "as static as possible", to work on a bigger variety of systems. Simply compiling like this will give a binary that is unlikely to work on the machine on most users, and thus mostly useless. I'll set up a proper release pipeline once i actually _do_ an official release ;D